### DISCUSSION TITLE
新找到的毒瘤，应该还没在反吸血名单里

### DISCUSSION THREAD

**Discussion by nulllee** [c0re100/qBittorrent-Enhanced-Edition#365] [General]
https://www.baidu.com/s?ie=utf-8&f=8&rsv_bp=1&tn=baidu&wd=rain%20bt%E4%B8%8B%E8%BD%BD%E5%99%A8
https://blog.csdn.net/weixin_34212189/article/details/85983312

今天下载东西的时候偶然发现有个叫rain 0.0.0的客户端，从我这下载的速度很快，用bitcomet看没有给别人上传，他自身进度一直是1.9%。上面是百度搜到的结果，是个国外的离线下载器。

**Comment by SeaHOH** [comment]
离线和种子盒之类的确有可能吸血，但你看到的好像是这个：
https://github.com/cenkalti/rain/blob/master/torrent/version.go

而且你链接里描述的那个离线网站已经消失了。

**Reply by nulllee** [reply]
谢谢解答。确实符合描述，可能是开发者还没做完导致功能有缺失。我一会自己删掉这个话题。

**Reply by SeaHOH** [reply]
它当前版本是 1.8.6，你看到的不是官方发布的正式版本。
也许有人改了个吸血的也说不定。不过，只要没大范围传播开就无所谓专门去针对屏蔽。

**Reply by nulllee** [reply]
> 它当前版本是 1.8.6，你看到的不是官方发布的正式版本。 也许有人改了个吸血的也说不定。不过，只要没大范围传播开就无所谓专门去针对屏蔽。

如此的话，我就不删除这个话题了，让作者自己定夺吧。我也是偶然发现的，大多数客户端都见过，这个rain 0.0.0很新鲜。
